volvo-cars/lingon
# Dashboards

This PR improves the ease of creating/maintaining dashboards.

To test it locally:
```bash
cd docs/platypus2/monitoring

go test -v . -keep-cluster=true -tags inttest
```
This will create a kind cluster and allow us to iterate rapidly. 

- run the test
- make changes
- re-run the test to deploy the changes to the same cluster for faster iteration.

Note that if `-keep-cluster=false`, the kind cluster will be delete as the test is quite basic for now (it is just checking if Grafana is starting successfully).


What is monitored:

- kubernetes (coredns, api-server, ...)
- grafana
- NATS
- karpenter
- victoria-metrics
